app/controllers/actions/AuthAction.scala

```diff
@@ -39,13 +39,13 @@ class AuthActionImpl @Inject()(override val authConnector: AuthConnector, config
   val ctUtr: Retrieval[Option[String]] = OptionalRetrieval("ctUtr", Reads.StringReads)
```

Comment:
Should argument be ctUtr or something related to vat? Not sure how it works or where it's used.